builtin/providers/azurerm/resource_arm_servicebus_topic.go

```diff
@@ -55,6 +55,7 @@ func resourceArmServiceBusTopic() *schema.Resource {
 			"duplicate_detection_history_time_window": {
 				Type:     schema.TypeString,
 				Optional: true,
+				Computed: true,
```

Comment:
Yeah - unfortunately setting this to `Optional: true, Default: "00:30:00"` leaves a diff in the Import test:
```
TF_ACC=1 go test ./builtin/providers/azurerm/ -v -run=TestAccAzureRMServiceBusTopic_import -timeout 120m
=== RUN   TestAccAzureRMServiceBusTopic_importBasic
--- FAIL: TestAccAzureRMServiceBusTopic_importBasic (349.48s)
	testing.go:280: Step 1 error: ImportStateVerify attributes not equivalent. Difference is shown below. Top is actual, bottom is expected.

		(map[string]string) {
		}


		(map[string]string) (len=1) {
		 (string) (len=39) "duplicate_detection_history_time_window": (string) (len=8) "00:30:00"
		}

FAIL
exit status 1
FAIL	github.com/hashicorp/terraform/builtin/providers/azurerm	349.497s
```